src/main/features/ccj/routes/paid-amount-summary.ts

```diff
@@ -6,17 +6,22 @@ import { ccjPath, Paths } from 'features/ccj/paths'
 import { DraftCCJ } from 'ccj/draft/draftCCJ'
 import { AbstractModelAccessor, DefaultModelAccessor } from 'shared/components/model-accessor'
 import { PaidAmount } from 'ccj/form/models/paidAmount'
+import { Claim } from 'claims/models/claim'
 
 class PaidAmountSummaryPage extends AbstractPaidAmountSummaryPage<DraftCCJ> {
 
-  createModelAccessor (): AbstractModelAccessor<DraftCCJ, PaidAmount> {
+  paidAmount (): AbstractModelAccessor<DraftCCJ, PaidAmount> {
     return new DefaultModelAccessor('paidAmount', () => new PaidAmount())
   }
 
   buildRedirectUri (req: express.Request, res: express.Response): string {
     const { externalId } = req.params
     return Paths.paymentOptionsPage.evaluateUri({ externalId: externalId })
   }
+
+  amountSettledFor (claim: Claim, draft: DraftCCJ): number {
+    return undefined
```

Comment:
Is this as expected? For CCJ we don't have a settlement amount?

Reply:
Yes, regular CCJ is a judgment by default for the full amount, whereas by admission you can settle for a lower amount